File: src/main/java/com/kyiv_kug/statics/usage/jv/Usage.java
package com.kyiv_kug.statics.usage.jv;

import com.kyiv_kug.statics.declaration.StaticsClassJava;
import com.kyiv_kug.statics.declaration.StaticsClassKotlin;

public class Usage {
    static void staticJava() {
        int CONST = StaticsClassJava.CONST;
        Object CONST_OBJ = StaticsClassJava.CONST_OBJ;

        StaticsClassJava.staticMethod();
    }

    static void staticKotlin() {
        int CONST = StaticsClassKotlin.CONST;
        Object CONST_OBJ = StaticsClassKotlin.CONST_OBJ;

        StaticsClassKotlin.staticMethod();
    }
}

File: src/main/java/com/kyiv_kug/statics/usage/kt/Usage.kt
package com.kyiv_kug.statics.usage.kt

import com.kyiv_kug.statics.declaration.StaticsClassJava
import com.kyiv_kug.statics.declaration.StaticsClassKotlin

fun staticClass() {
    val CONST = StaticsClassJava.CONST
    val CONST_OBJ = StaticsClassJava.CONST_OBJ

    StaticsClassJava.staticMethod()
}

fun staticClassKotlin() {
    val CONST = StaticsClassKotlin.CONST
    val CONST_OBJ = StaticsClassKotlin.CONST_OBJ

    StaticsClassKotlin.staticMethod()
}